docs/schema/validation.json

```diff
@@ -0,0 +1,53 @@
+{
+  "$schema": "https://json-schema.org/draft-07/schema",
+  "title": "Validation settings",
+  "markdownDescription": "https://www.mkdocs.org/user-guide/configuration/#validation",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "omitted_files": { "$ref": "#/$defs/omitted_files" },
+    "not_found": { "$ref": "#/$defs/not_found" },
+    "absolute_links": { "$ref": "#/$defs/absolute_links" },
+    "nav": {
+      "title": "validation of navigation",
+      "markdownDescription": "https://www.mkdocs.org/user-guide/configuration/#validation",
+      "additionalProperties": false,
+      "properties": {
+        "omitted_files": { "$ref": "#/$defs/omitted_files" },
+        "not_found": { "$ref": "#/$defs/not_found" },
+        "absolute_links": { "$ref": "#/$defs/absolute_links" }
+      }
+    },
+    "links": {
+      "title": "Validation of links",
+      "markdownDescription": "https://www.mkdocs.org/user-guide/configuration/#validation",
+      "additionalProperties": false,
+      "properties": {
+        "not_found": { "$ref": "#/$defs/not_found" },
+        "absolute_links": { "$ref": "#/$defs/absolute_links" },
+        "unrecognized_links": { "$ref": "#/$defs/unrecognized_links" }
+      }
+    }
+  },
+  "$defs": {
```

Comment:
Please use `"definitions"` and not `$defs`, as we do in the rest of our schema, e.g. here:

https://github.com/squidfunk/mkdocs-material/blob/57e598b96e8a976eec0559de99ab700bf2d6c38e/docs/schema/theme.json#L457-L468

Reply:
You're right, I didn't know that 🤦‍♂️

> Although we can identify any subschema using JSON Pointers or named anchors, the $defs keyword gives us a standardized place to keep subschemas intended for reuse in the current schema document.

Could you open another PR then, renaming all `definitions` to `$defs`? Consistency is really important.